app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,73 @@
+
 .check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+  > * {
```

Comment:
FYI, I did it this way to allow other markup than a description list to use the same CSS. E.g. you could use a list or some divs instead and it would still look and behave the same.
I know this can also be done by giving every container and every item a class. I suspect that might be what people will ask me to change. I personally prefer it the way it is right now but would be fine with changing it if people felt strongly about it.